packages/atomic/src/components/facets/facet-common.ts

```diff
@@ -23,3 +30,68 @@ export interface FacetValueProps {
   part?: string;
   buttonRef?: (element?: HTMLButtonElement) => void;
 }
+
+function isCategoryFacetValueRequest(
+  value: AnyFacetValueRequest
+): value is CategoryFacetValueRequest {
+  return 'children' in value && Array.isArray(value.children);
+}
+
+function isFacetValueRequest(
+  value: AnyFacetValueRequest
+): value is FacetValueRequest | CategoryFacetValueRequest {
```

Comment:
Why `FacetValueRequest | CategoryFacetValueRequest` looks like it filters out values with `children`
```suggestion
): value is FacetValueRequest {
```

Reply:
Woops, you're reight.